drift/info_parser.py

```diff
@@ -190,9 +190,11 @@ def _group_comparisons(comparisons):
         group["comparisons"] = sorted(
             group["comparisons"], key=lambda comparison: comparison_name
         )
+
+    grouped_comparisons = grouped_comparisons + list(grouped_comparisons_with_dotnotation.values())
     # Need to add all dot notation comparisons into the output grouped_comparisons list
-    for comparison in grouped_comparisons_with_dotnotation:
-        grouped_comparisons.append(grouped_comparisons_with_dotnotation.get(comparison))
+    # for comparison in grouped_comparisons_with_dotnotation:
+    #     grouped_comparisons.append(grouped_comparisons_with_dotnotation.get(comparison))
```

Comment:
Please remove these. I suggest to fixup this change to the corresponding commit: "Using dict instead of list".

Reply:
fixed